accepted/targeting-packs/targeting-packs.md

```diff
@@ -0,0 +1,149 @@
+the corresponding targeting pack.
+
+Since .NET Framework 2.0, the .NET platform supported targeting different
+versions of the platform regardless of the .NET Framework version that is
+installed on the developer's machine. This allows developers to target a lower
+version than what they use themselves. This is especially useful for in-place
+updates where the older version of the framework cannot be installed side-by-
+side.
+
+That mechanism is called *targeting packs*, which is simply a set of assemblies
+that describe the public APIs of a particular version of the .NET Framework.
+Since those assemblies are only used by the compiler, they don't need to contain
+any code, which we call [reference assemblies](https://www.youtube.com/watch?v=EBpY1UMHDY8).
+
+Unfortunately, the targeting packs aren't available by themselves; they are
+redistributed as part of the .NET Framework Developer Packs which bundle both
+the runtime as well as the targeting pack. This design ensures that developers
+can never be in a state where they have a targeting pack that is higher than the
+.NET Framework version they have installed. That design made sense coming from a
+Visual Studio-first perspective but doesn't serve us well today:
+
+* **It's not cross-platform friendly**. The .NET Framework Developer Packs are
+  only available for Windows which makes the targeting packs indirectly Windows-
+  only too.
+
+* **It's not CI friendly**. Requiring a machine-wide install isn't friendly for
+  cloud-hosted CI machines as developers typically lack permissions to install
+  software.
+
+This document proposes to offer the targeting packs via NuGet that they can be
+acquired in a non-impactful way. It builds on the following existing specs:
+
+* [Phillip Carter](https://github.com/cartermp): [Add Target Packs on NuGet](https://github.com/dotnet/core/pull/64)
+* [Daniel Plaisted](https://github.com/dsplaisted): [Reference Assembly NuGet packages](https://gist.github.com/dsplaisted/83d67bbcff9ec1d0aff1bea1bf4ad79a)
+* [Prototype](https://github.com/dsplaisted/ReferenceAssemblyPackages)
+
+## Scenarios and User Experience
+
+* Build a project which is multi-targeted to .NET Core or .NET Standard as well
+  as .NET Framework on Mac OS or Linux using the .NET CLI
+
+* Build a project targeting Mono (which uses the same Target Framework
+  Identifier as .NET Framework) using the .NET CLI
+
+## Requirements
+
+### Goals
+
+* Doesn't require running an installer
+* Doesn't require Windows
+* Can be used in SDK-style multi-targeting projects
+
+### Non-Goals
+
+* Supporting NuGet-based acquisition of the .NET Framework runtime
+* Supporting NuGet-based acquisition from non-SDK-style projects
+* Support pre-.NET Framework 4.5 targeting packs
+* Support non-.NET Framework targeting packs (Xamarin, Mono, Unity)
+
+## Design
+
+* Daniel has created a [prototype](https://github.com/dsplaisted/ReferenceAssemblyPackages).
+    - We have an identity package
```

Comment:
It's not clear to me what this means. It may not be important for it to be clear - that depends on the audience and potential longevity of this doc.

There are a few other bits that I could suggest clarifications/fixes to, but they don't have a technical impact. Let me know if you want nit-picking on the wording, basically. Otherwise, I'll assume that as a proposal doc, this is effectively transient.

Reply:
"Identity package" is a term we've used internally. The basic idea is that you a single package ("the identity") that is empty and only depends on other packages split by TFM version. It's basically a very specific usage of a meta package that aggregates a bunch of packages under a common name. I've reworded the entire section to make it easier to understand.